app/views/compute_resources_vms/form/vmware/_base.html.erb

```diff
@@ -13,6 +13,8 @@
 <%= select_f f, :guest_id, compute_resource.guest_types, :first, :last, {}, { :label => _("Guest OS"), :class => "col-md-2", :disabled => !new_host } %>
 <%= select_f f, :scsi_controller_type, compute_resource.scsi_controller_types, :first, :last, {}, { :label => _("SCSI controller"), :class => "col-md-2", :disabled => !new_host } %>
 <%= select_f f, :hardware_version, compute_resource.vm_hw_versions, :first, :last, {}, { :label => _("Virtual H/W version"), :class => "col-md-2", :disabled => !new_host } %>
+<%= checkbox_f f, :memoryHotAddEnabled, { :help_inline => _("Enable memory hot add"), :label => _('Memory hot add'), :label_size => "col-md-2", :disabled => !new_host } if new_host %>
```

Comment:
Ah great.  Now I see what it looks like, you might want to get rid of the help_inline, since it's just a copy of the text - unless there's something more to say.